allure-ruby-commons/lib/allure_ruby_commons/model/parameter.rb

```diff
@@ -3,13 +3,25 @@
 module Allure
   # Allure model parameter object
   class Parameter < JSONable
-    def initialize(name, value)
+    MODES = %w[default masked hidden].freeze
```

Comment:
I would define explicit constants, as it makes a lot easier when using it in the code and makes it easier to rename if for some reason naming convention changes.

```suggestion
    DEFAULT = "default".freeze
    MASKED = "masked".freeze
    HIDDEN = "hidden".freeze

```